elements/lisk-dpos/src/delegates_info.ts

```diff
@@ -366,8 +366,8 @@ export class DelegatesInfo {
 			uniqDelegateListWithRewardsInfo,
 			totalFee,
 		} = blocksInRounds.reduce(
-			(acc: AccountSummary, fetchedBlock: Block, i) => {
-				acc.totalFee = acc.totalFee.add(fetchedBlock.totalFee);
+			(acc: AccountSummary, fetchedBlock: Block | BlockJSON, i) => {
+				acc.totalFee = acc.totalFee + BigInt(fetchedBlock.totalFee);
```

Comment:
```suggestion
				acc.totalFee = BigInt(acc.totalFee) + BigInt(fetchedBlock.totalFee);
```

Reply:
`totalFee` has `add` meaning that it's already BigNum. therefore, no need to double cast